avocado/core/remote/runner.py

```diff
@@ -264,6 +265,13 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
             self.result.end_tests()
             self.job._result_events_dispatcher.map_method('post_tests',
                                                           self.job)
+            # compare the major version of remote and local version of avocado. If different write to UI and job.log
+            if str(remote_version[0]) != version.MAJOR:
+                log = logging.getLogger("avocado.job")
```

Comment:
I agree with using `APP_LOG` & `TEST_LOG` it adds consistency between the two `runner.py` files. I'll change that in my v4 branch